Dockerfile

```diff
@@ -2,63 +2,42 @@ FROM alpine:3.7
 
 MAINTAINER Andy Driver <andy.driver@digital.justice.gov.uk>
 
-# install build dependencies (they'll be uninstalled after pip install)
-RUN apk add --no-cache \
-        --virtual build-deps \
-        gcc \
-        musl-dev
+ENV HELM_VERSION 2.9.1
+ENV HELM_HOME /tmp/helm
+ENV DJANGO_SETTINGS_MODULE "control_panel_api.settings"
+ENV USE_VENV=false
```

Comment:
Why there is this flag? Maybe it's a stupid question but why someone would want to enable/disable this while running the Docker image?

Reply:
This is because if you run `make dependencies` outside of the docker container, it will use a virtualenv by default. In the docker container, we don't want to use a virtualenv, so this environment variable prevents that.

Reply:
I've removed this now, because I removed the `make` dependency in the docker image.

Reply:
Thanks for the clarification @andyhd 👍 

Just out of curiosity, why we don't want to use virtualenv in the Docker container? What are the drawbacks of that?

Isn't virtualenv pretty much creating a glorified simlink to a directory? That should't have much of a performance hit - but maybe I'm oversimplifying here.

Reply:
Mainly for convenience - if you want to run `manage.py` for example inside the docker container, you would have to write `docker run -it control-panel venv/bin/python3 manage.py shell`. A virtualenv is unnecessary inside a docker container, so why create it in the first place?

Reply:
Oh of course! Unless we don't symlink that `venv/bin/python3` binary. But you're right, I see why it's more convenient this way 👍